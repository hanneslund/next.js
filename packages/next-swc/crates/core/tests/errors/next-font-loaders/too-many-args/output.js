import inter from "@next/font/google?Inter;{}";
