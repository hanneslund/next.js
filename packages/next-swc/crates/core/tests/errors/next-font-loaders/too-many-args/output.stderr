
  x Font loaders only accepts 1 argument
   ,-[input.js:2:1]
 2 | const inter = Inter({}, 10)
   :               ^^^^^^^^^^^^^
   `----
